benchmark/Benchmark/Htoml.hs

```diff
@@ -17,10 +18,8 @@ import Data.Text (Text)
 import GHC.Generics (Generic)
 import Text.Parsec.Error (ParseError)
 
-import Text.Toml (parseTomlDoc)
-import Text.Toml.Types (Node, Table)
-
-import qualified Text.Toml.Types as TTT
+import "htoml" Text.Toml (parseTomlDoc)
+import "htoml" Text.Toml.Types (Node (..), Table, toJSON)
```

Comment:
Yeah, that's a problem with module names. But `PackageImports` is a proper solution to this problem :+1: